helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -78,6 +83,13 @@ public enum RebalanceStatus {
   // Histograms
   private HistogramDynamicMetric _partitionTopStateHandoffDurationGauge;
   private HistogramDynamicMetric _partitionTopStateHandoffHelixLatencyGauge;
+  // A new Gauage for reporting number of partitions with missing top state has been added. The main reason of deprecating
+  // this gauge because it's similar to other failedTopStateCounter and averaging out duration of all those handsoff
+  // won't really help in debugging the issue. Hence please use new gauage _missingTopStatePartitionsBeyondThresholdGauge
+  // to find number of partitions with missing top state beyond threshold and to find duration use
+  // _maxSinglePartitionTopStateHandoffDuration. New gauage will help with that. Please find more info at
+  // https://github.com/apache/helix/pull/2381
+  @Deprecated
```

Comment:
No need to add duplicate comments.

Reply:
It's not duplicate, the reason to deprecate  _partitionTopStateNonGracefulHandoffDurationGauge is slightly different than other one.

Reply:
Thanks for the reply. Please consider using more succinct wording.